mathoxide-lib/src/array.rs

```diff
@@ -1,15 +1,110 @@
 use crate::storage::Storage;
+use crate::views;
+use crate::views::ArrayView;
 
-pub struct Array<StorageType> {
+use num_traits::Num;
+
+use std::fmt;
```

Comment:
I prefer to have the `use` statements ordered by `std`, then external, then internal.